### PR TITLE
add Oasis in supported blockchains list

### PR DESCRIPTION
**Pull request by babybunn**
Screen Shot 2565-02-28 at 14 55 59

### CODE REVIEW

File: docs/band-standard-dataset/supported-blockchains.md [hunk 1]
```diff
@@ -12,14 +12,15 @@ We recommend experimenting with using our solution on a test network first befor
...
-| ---------------- | :----------------------------------------: | :-----------------------------------------------------------------------------------: |
-| Ethereum (Kovan) | 0xDA7a001b254CD22e46d3eAB04d937489c93174C3 | [link](https://kovan.etherscan.io/address/0xDA7a001b254CD22e46d3eAB04d937489c93174C3) |
+| Blockchain       |      `StdReference` Contract Address       |                                                 Explorer                                                 |
+| ---------------- | :----------------------------------------: | :------------------------------------------------------------------------------------------------------: |
+| Ethereum (Kovan) | 0xDA7a001b254CD22e46d3eAB04d937489c93174C3 |          [link](https://kovan.etherscan.io/address/0xDA7a001b254CD22e46d3eAB04d937489c93174C3)           |
 | Celo (Alfajores) | 0x71046b955Cdd96bC54aCa5E66fd69cfb5780f3BB | [link](https://alfajores-blockscout.celo-testnet.org/address/0x71046b955Cdd96bC54aCa5E66fd69cfb5780f3BB) |
+| Oasis            | 0xDA7a001b254CD22e46d3eAB04d937489c93174C3 |                               [link](https://explorer.emerald.oasis.dev/)                                |
```
Comment (p-rial) [inline docs/band-standard-dataset/supported-blockchains.md]: image

In what condition we will add name of testnet environment ?

**Review by p-rial — CHANGES_REQUESTED** [review]

File: docs/band-standard-dataset/supported-blockchains.md (hunk 1 above)
Comment (babybunn) [reply docs/band-standard-dataset/supported-blockchains.md]: Sorry, I missed it. The testnet env should be specified if they have more than one environtments.

**Comment by p-rial** [comment]
😎 Browse the preview: https://deploy-preview-63--docs-staging.netlify.app/
@babybunn The link above should show the result of this PR, right ? 
image

**Review by p-rial — APPROVED** [review]

**Comment by p-rial** [comment]
@babybunn Once this changed deploy on production, notice me again pls. I need to notify partner

**Pull request merged**